### ISSUE TITLE
IIIF manifest should include item metadata

### ISSUE THREAD

**Issue by ebenenglish** [marriott-library/newspaper_works#72]
**Descriptive Summary**
The IIIF manifest for a Newspaper object (Issue, Page, Article) should include descriptive information about the item represented by the manifest.

**Rationale**
Since IIIF manifests are often used to present objects outside of their "home" repository context, including descriptive metadata (title, creator, etc.) helps provide useful information about the item.

**Expected behavior**
As a user
When I view the IIIF manifest for a NewspaperIssue
The "metadata" attribute should contain an array of properties 
So the descriptive information about the item is connected to its image content

**Additional info**
The presence of the metadata is set in `config/initializers/hyrax.rb` in the local application. By default, this is set to:
```ruby
config.iiif_metadata_fields = Hyrax::Forms::WorkForm.required_fields
```
We will need to create a generator to insert our fields into the `iiif_metadata_fields` array, in a way that is not disruptive to other objects in the local application.

This could potentially be an additional line, like so:
```
config.iiif_metadata_fields = (config.iiif_metadata_fields + NewspaperWorks::NewspaperCoreFormData.required_fields).uniq
```
This requires that the `Presenter` object for the work can `respond_to?` each metadata property in `NewspaperWorks::NewspaperCoreFormData.required_fields`, which is not currently available.

Ongoing work for #45 should make this functionality possible.

**This ticket will be complete when:**
Item metadata is present in the manifest

**Comment by ebenenglish** [comment]
Probably easier to simply overwrite `#manifest_metadata` in the presenter class for Issue, Page, and Article objects. A one-size-fits all approach will be difficult since the metadata fields are significantly different for each of these object types.

See: https://github.com/samvera/hyrax/blob/2.x-stable/app/presenters/hyrax/work_show_presenter.rb#L216-L225

Hold off on this until #141 is merged, since there are a lot of Presenter changes in that PR, which could make merging a pain.

**Comment by ebenenglish** [comment]
Closed via #144.
